File: ModelInterfacesClassLibrary/Phonebook/LprEmployee01/LprEmployee01ViewPage.interface.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using Newtonsoft.Json.Serialization;
using ModelInterfacesClassLibrary.Phonebook.LprEmployee01;

namespace ModelInterfacesClassLibrary.Phonebook.LprEmployee01 {
    public interface ILprEmployee01ViewPage
    {
        [JsonProperty(PropertyName = "page")]
        int page { get; set; }

        [JsonProperty(PropertyName = "pagesize")]
        int pagesize { get; set; }

        [JsonProperty(PropertyName = "pagecount")]
        int pagecount { get; set; }

        [JsonProperty(PropertyName = "total")]
        int total { get; set; }

        [JsonProperty(PropertyName = "items")]
        IList<ILprEmployee01View> items { get; set; }
    }
}

File: AngularMatPhonebookSolution/projects/phonebook-interfaces/src/lib/lpr-employee01/interfaces/lpr-employee01-view-page.interface.ts
import { ILprEmployee01View } from './lpr-employee01-view.interface';

export interface ILprEmployee01ViewPage {
        page : number; // int
        pagesize : number; // int
        pagecount : number; // int
        total : number; // int
        items ?: ILprEmployee01View[];
}


